tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/HotChocolate/IOperationProxy.cs

```diff
@@ -1,24 +1,25 @@
-// <copyright file="PreparedOperationStruct.cs" company="Datadog">
+// <copyright file="IOperationProxy.cs" company="Datadog">
 // Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
+#nullable enable
+
 using Datadog.Trace.DuckTyping;
 
 namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.GraphQL.HotChocolate
 {
     /// <summary>
-    /// HotChocolate.Execution.Processing.IPreparedOperation interface for ducktyping
+    /// HotChocolate.Resolvers.IOperation interface for ducktyping
     /// </summary>
-    [DuckCopy]
-    internal struct PreparedOperationStruct
+    internal interface IOperationProxy
     {
```

Comment:
changed this because IPreparedOperation implements IOperation, and the two ducktyped properties are in the IOperation interface, so it makes more sense to mention `IOperation` here I think.